inappmessaging/src/main/java/com/rakuten/tech/mobile/inappmessaging/runtime/data/repositories/ConfigResponseRepository.kt

```diff
@@ -44,20 +47,29 @@ internal interface ConfigResponseRepository {
          */
         @VisibleForTesting
         fun resetInstance() {
+            randomizer = Random
             this.instance = ConfigResponseRepositoryImpl()
         }
     }
 
     private class ConfigResponseRepositoryImpl : ConfigResponseRepository {
         private var configResponseData: ConfigResponseData? = null
+        private var isEnabled = false
 
         @Throws(IllegalArgumentException::class)
+        @SuppressWarnings("MagicNumber")
         override fun addConfigResponse(data: ConfigResponseData?) {
             requireNotNull(data)
             configResponseData = data
+            val rollOut = configResponseData?.rollOutPercentage ?: 0
+            isEnabled = if (rollOut > 0) {
+                randomizer.nextInt(1, 101) <= rollOut
```

Comment:
Is there a difference between `(1..100).random()` and `Random.nextInt(1, 101)`?

Reply:
No difference in behavior. But `Random` is easier to mock.